packages/dds/tree/src/feature-libraries/modular-schema/fieldChangeHandler.ts

```diff
@@ -90,7 +90,9 @@ export interface FieldEditor<TChangeset> {
     buildChildChange(childIndex: number, change: NodeChangeset): TChangeset;
 }
 
-export type ToDelta = (child: NodeChangeset) => Delta.Modify;
+export type ToDelta = (child: NodeChangeset, index: number | undefined) => Delta.Modify;
+
+export type RepairData = (revision: RevisionTag, index: number, count: number) => Delta.ProtoNode[];
```

Comment:
I was surprised this was a function type. I don't think we've settled on any conventions for naming these, but something like `NodeRecoverer` would be more consistent with other delegate types in this file, and in my opinion would make it easier to read usages of this type.

Reply:
I went for `NodeReviver` which rolls of the tongue a little better. Happy to workshop the name some more.